tests/test_doc_examples.py

```diff
@@ -12,7 +13,8 @@ def _get_coding_blocks(nb_file):
 @pytest.mark.parametrize(
     ('nb_file', 'idx_execute'),
     (
-        pytest.param('examples.buildcell.ipynb', 'all', id='buildcell'),
+        pytest.param('examples.buildcell.ipynb', 'all', id='buildcell',
```

Comment:
The existing `buildcell` test uses a different env var, but I agree that there's no easy way to make this example test use it. Can you change the existing `buildcell` test to fall back to searching in PATH (like this example test) if `WFL_PYTEST_BUILDCELL` isn't set?

Reply:
I changed the tests to look for both WFL_PYTEST_BUILDCELL and in PATH.

When I was looking into the buildcell-tests, I was wondering about the remote case since the path to the buildcell-executable on the cluster might differ from the one on a local machine. Is this something to worry about?

When running it via `pytest test_buildcell.py --runremote --basetemp $HOME/pytest` the tests run and pass on my local machine, however, jobs are also not send to the cluster. Is this an intended behavior?

Reply:
> When I was looking into the buildcell-tests, I was wondering about the remote case since the path to the buildcell-executable on the cluster might differ from the one on a local machine. Is this something to worry about?

In some sense yes, but given that the remote machine can be arbitrarily different from the local machine, there's not going to be a general solution.  I'm happy with requiring that either the absolute path be the same, or that it be in `PATH` (and therefore only need the executable name). If needed we can add support for a pytest-wide customization of the remote job `PATH` (e.g. by having pytest add an `env_var` for all its remote jobs).

I think it'd be too hard to make the system be entirely agnostic, in a completely general way, as to what machine it's running on. I can live with the current situation, i.e. that if the user specifies a remote job, they have to make sure the paths they pass are the correct ones for the remote machine.  It's not optimal for testing, but I think that for real production runs, if you're using a script you can always make the path to things like buildcell come from  a variable, which you can set at runtime.

Reply:
I see. Thanks for clarifying.